grow/pods/podcache.py

```diff
@@ -41,12 +55,59 @@ def document_cache(self):
         """Cache for specific document properties."""
         return self._document_cache
 
+    @property
+    def object_cache(self):
+        """Global object cache."""
+        return self.get_object_cache(self.KEY_GLOBAL)
+
+    def create_object_cache(self, key, write_to_file=False, can_reset=True, values=None):
```

Comment:
Why does the object cache have a distinction for writing to a file, but the others don't? Should we just make the podcache always used for persistent (file) storage and not have the user have to make a decision about it?